antlr-kotlin-runtime/src/nativeMain/kotlin/com/strumenta/antlrkotlin/runtime/WeakHashMap.kt

```diff
@@ -15,6 +15,5 @@
  */
 package com.strumenta.antlrkotlin.runtime
 
-public actual class WeakHashMap<K, V>(private val wrapped: LinkedHashMap<K, V> = LinkedHashMap()) : MutableMap<K, V> by wrapped {
-  public actual constructor() : this(LinkedHashMap())
-}
+// TODO(Edoardo): implement real weak keys
```

Comment:
Would that even be possible?

Reply:
Yes, see this package https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.native.ref/

Requires more time than what I have now, that's why I postponed it

Reply:
Ok, could you add this link into this comment then?